libc/src/__support/HashTable/generic/bitmask_impl.inc

```diff
@@ -51,6 +51,11 @@ struct Group {
     return {data.value};
   }
 
+  // Load a group of control words from an aligned address.
+  LIBC_INLINE static Group load_aligned(const void *__restrict addr) {
+    return *static_cast<const Group *>(addr);
+  }
```

Comment:
Why use `__restrict` here? What are you trying to avoid aliasing with?

Reply:
Removed in the latest commit.